.gitignore

```diff
@@ -12,6 +12,7 @@
 **/*.stories.json
 **/node_modules/
 *.log
+*.orig
```

Comment:
What's this extension?

Reply:
that's a file that sometimes is generated by my difftool

Reply:
Ideally you'd handle this in a global .gitignore on your machine, but I'm cool merging this in ❤️